src/lib/openapi/spec/proxy-client-schema.ts

```diff
@@ -22,6 +23,8 @@ export const proxyClientSchema = {
         environment: {
             type: 'string',
             deprecated: true,
+            description: 'deprecated',
+            example: 'deprecated',
```

Comment:
Could probably use a standard env name here, but `deprecated` works too. Your call.
```suggestion
            example: 'development',
```